### PR TITLE
change/navarbar-font

### PR DESCRIPTION
**Pull request by Kendiya242**
Hello team i have change the navbar font and size

### CODE REVIEW

**Review by dinaolmelak — CHANGES_REQUESTED** [review]
Please double check if your code has changed anything.

File: src/component/navbar/Navbar.jsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ export default function NavBar() {
   };
 
   return (
-    <nav className={styles.nav}>
+    <nav className={styles.nav}>
```
[the remark this answers is not in the document]
Comment (Kendiya242) [reply src/component/navbar/Navbar.jsx:14]: ok I will double check and also can you see that the font is change ?

File: src/component/navbar/navbar.module.scss [hunk 2]
```diff
@@ -24,7 +24,7 @@ ul {
 }
 
 li {
-  margin: 0 1rem;
+  margin: 0 1rem;
```
[the remark this answers is not in the document]
Comment (Kendiya242) [reply src/component/navbar/navbar.module.scss:27]: I didnt add any changes to the margin

**Comment by dinaolmelak** [comment]
yap! Perfect!, lol even tho we define moving differently, you understood me 😅

File: src/component/navbar/Navbar.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Kendiya242) [reply src/component/navbar/Navbar.jsx:14]: there wasnt no changes done to the Navbar.jsx only the module.scss. What i did is added the font family, weight, size and style to a { class in the navbar.module.scss to make the font visible. So the  className={styles.a} in Navbar.jsx defines the class a { in the module. scss which adds the styling of the font.

[the remark this answers is not in the document]
Comment (dinaolmelak) [reply src/component/navbar/Navbar.jsx:14]: @Kendiya242 , you have two PRs do you see it? Why not close this PR out, Delete this branch and then continue to work with the other branch. Cause there are no changes on this PR.

[the remark this answers is not in the document]
Comment (Kendiya242) [reply src/component/navbar/Navbar.jsx:14]: after i make the changes on this PR and push i will close this one

File: src/component/navbar/navbar.module.scss [hunk 3]
```diff
@@ -25,13 +26,16 @@ ul {
...
 li {
-  margin: 0 1rem;  
+  margin: 0 2rem;  
 }
 
 a {
   color: black; 
   text-decoration: none;
-  font-size: 1rem;
+  font-size: 25px;
```
Comment (dinaolmelak) [inline src/component/navbar/navbar.module.scss:35]: sorry @Kendiya242 , I didn't see this specific change that altered the navbar styling and you didn't specify which lines of code you changed hence I was under the assumption that there were not any changes but minor. Considering that most of the files just had spaces I did not interrogate closer.

**Review by dinaolmelak — APPROVED** [review]

File: src/component/navbar/navbar.module.scss (hunk 3 above)
Comment (Kendiya242) [reply src/component/navbar/navbar.module.scss:35]: its ok next time I will state which specific changes I made on a next PR.

**Pull request merged**